### PR TITLE
DCOS-38652: update to 3.1.0

### PR DESCRIPTION
**Pull request by nLight**
If CI is green it should be good to go!

Closes DCOS-38652

## Testing

1. Pull
2. npm install
3. npm run build-assets
4. npm run integration-tests
5. Maybe run system tests locally as well

### CODE REVIEW

**Comment by nLight** [comment]
They've changed the way Cypress caches the binary we have to adjust our isolation setup.

**Comment by juliangieseke** [comment]
System Tests are red 😭

**Comment by nLight** [comment]
@juliangieseke yes, They've changed the way Cypress caches the binary we have to adjust our isolation setup.

File: system-tests/cypress.json [hunk 1]
```diff
@@ -9,6 +9,5 @@
   },
   "videoUploadOnPasses": false,
   "numTestsKeptInMemory": 0,
-  "screenshotOnHeadlessFailure": false,
```
[the remark this answers is not in the document]
Comment (nLight) [reply system-tests/cypress.json]: Deprecated. That was a bad idea anyway. Screenshots are helpful

[the remark this answers is not in the document]
Comment (juliangieseke) [reply system-tests/cypress.json]: yeah, was wondering why that line was in the file, not why you removed it 👍

**Pull request merged**